README.md

```diff
@@ -337,6 +344,90 @@ spec:
 - The operator fetches the secret from GCP Secret Manager and injects it as a
 secret:
 
+```shell
+% make install
+% make deploy
+% kubectl get secret externalsecret-operator-externalsecret-sample -n externalsecret-operator-system \
+  -o jsonpath='{.data.example-externalsecret-key}' | base64 -d
+```
+
+<a name="gitlab-cicd-variables"></a>
+
+## Gitlab CI/CD Variables
+
+<a name="gitlab-cicd-variables-pre"></a>
+
+#### Prerequisites
+- A Gitlab project with a CI/CD variable, who's key is `example-externalsecret-key`
+- A [Gitlab personal access token](https://gitlab.com/-/profile/personal_access_tokens)
+
+- Install CRDs
+```
+  make install
+```
+
+<a name="gitlab-cicd-variables-deployment"></a>
+
+#### Deployment
+
+- Uncomment and update credentials to be used in `config/credentials/kustomization.yaml`:
+
+```yaml
+resources:
+# - credentials-gsm.yaml
+# - credentials-asm.yaml
+# - credentials-dummy.yaml
+# - credentials-onepassword.yaml
+- credentials-gitlab.yaml
+```
+
+- Update the gitlab credentials `config/credentials/credentials-gitlab.yaml` with your personal access token
+
+```yaml
+%cat config/credentials/credentials-gitlab.yaml
+...
+credentials.json: |-
+    {
+      "token": "abcdef12345"
+    }
+
+```
+-  Update the `SecretStore` resource definition `config/samples/store_v1alpha1_secretstore.yaml`
+```yaml
+% cat  `config/samples/store_v1alpha1_secretstore.yaml
+apiVersion: store.externalsecret-operator.container-solutions.com/v1alpha1
+kind: SecretStore
+metadata:
+  name: secretstore-sample
+spec:
+  controller: staging
+  store:
+    type: gitlab
+    auth:
+      secretRef:
+        name: externalsecret-operator-credentials-gitlab
+    parameters:
+      baseURL: https://gitlab.com
+      projectID: 12345678
```

Comment:
Could you document that the user also needs the Project ID and where to find it, on the Prerequisites section.

Thanks!